src/main/java/lt/codeacademy/zydrunasn/uzd1/SecretSanta.java

```diff
@@ -0,0 +1,29 @@
+package lt.codeacademy.zydrunasn.uzd1;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.Random;
+
+public final class SecretSanta {
+    public void peopleToExchangeGifts() throws IOException {
+        List<String> people = fileReader();
+        System.out.printf("%s -> %s",
+                randomPersonName(people),
```

Comment:
Pagal šitą implementaciją žmogus gali gauti dovanoti dovaną sau, ir vienas žmogus gali gauti kelias dovanas :)